### PR TITLE
feat(hardware-software): Increase sqlversion to 2017

### PR DESCRIPTION
**Pull request by pabloalonso**

### CODE REVIEW

**Review by baptistemesta — APPROVED** [review]

File: md/hardware-and-software-requirements.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ Software required for Bonita Platform (Bonita Engine and Bonita Portal).
 | **Database**                       | (see note 2)                                     |
 | MySQL                              | 8.0.14 and higher in the 8.0.x line (see note 3) |
 | PostgreSQL                         | 11.2 and higher in the 11.x line                 |
-| SQL Server                         | 2016                                             |
+| SQL Server                         | 2017                                             |
```
Comment (educhastenier) [inline md/hardware-and-software-requirements.md:36]: @pabloalonso Should we also have a specific focus on this update in RN page?

Comment (pabloalonso) [reply md/hardware-and-software-requirements.md:36]: it should be

Comment (educhastenier) [reply md/hardware-and-software-requirements.md:36]: Ok, so i'll let you complete this PR, ok?

Comment (pabloalonso) [reply md/hardware-and-software-requirements.md:36]: It is in another PR
https://github.com/bonitasoft/bonita-doc/pull/1245

**Pull request merged**